resources/lib/connectivity/cachehttpadapter.py

```diff
@@ -122,6 +122,8 @@ def __extract_cache_data(self, headers):
 
         if "cache-control" in headers:
             cache_control = headers['cache-control']
+            #The "Headliner" NRK API endpoint uses ';' instead of ',' as a field delimiter
+            cache_control = cache_control.replace(";", ",")
```

Comment:
I don't like this change. They are not sticking to the HTTP specifications: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#syntax

> Multiple directives are permitted and must be comma-separated (e.g., `Cache-control: max-age=180, public`).

Reply:
But I do see that things break now......

Reply:
Neither do I. I already wrote them an email. Unfortunately without this "fix" caching doesn't work and throws exceptions.

Reply:
Sure, I can change the comment to something more generic.